ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/CompletionDeduplicationInfoIT.scala

```diff
@@ -49,7 +49,6 @@ final class CompletionDeduplicationInfoIT[ServiceRequest](
       optSubmissionIdSubmittedCompletion <- service
         .submitRequest(ledger, party, requestWithSubmissionId)
     } yield {
-      assertApplicationIdIsPreserved(ledger.applicationId, optNoDeduplicationSubmittedCompletion)
```

Comment:
Why don't we need it?

Reply:
My thoughts are that this type of assertion is too low level for conformance tests, it should be part of participant integration tests. 

More than happy to revert the change if you think it makes sense.

Reply:
Sounds good, I also think some of the assertions are too detailed for conformance tests, as they are e2e/functional tests.

Reply:
I'd make this decision in another PR.